### ISSUE TITLE
[Feature Request] Update app icon to match MacOS 11+ style

### ISSUE THREAD

**Issue by ayndqy** [usebruno/bruno#1059]
It is worth making the icon of the program according to the style of MacOS 11 and later versions.

**Comment by MichaelCharles** [comment]
I agree. The current icon looks really out of place on the dock. Having an icon that looks like a modern app helps inspire confidence in the product, unfortunately. Shouldn't be that way but it is. Even just giving it a white background I think goes a long way to making it look more modern. 

<img width="1352" alt="Screenshot 2023-11-27 at 11 48 20" src="https://github.com/usebruno/bruno/assets/14105176/d4f09b5d-15c4-48e5-91bc-24e886aa8778">

**Comment by bpoulaindev** [comment]
For some reason i don't have the white background 
<img width="1458" alt="image" src="https://github.com/usebruno/bruno/assets/64689165/c85c493e-acd0-4115-b0fb-1e936f625b99">

**Comment by MichaelCharles** [comment]
> For some reason i don't have the white background 

Oh, no. That's just something I threw together for myself.

![bruno](https://github.com/usebruno/bruno/assets/14105176/9cb25b44-05fd-4623-8928-5894e3772115)

Here it is as an ICNS file.

[bruno.icns.zip](https://github.com/usebruno/bruno/files/13473088/bruno.icns.zip)

It isn't really anything special. I just think it looks slightly better.

If you right click on bruno.app and select "Get Info" a dialog box will pop up. If you drag the PNG or ICNS file onto the icon inside the dialog box, it will replace the icon.

**Comment by ayndqy** [comment]
Merged into main
